src/test/java/org/elasticsearch/test/ElasticsearchIntegrationTest.java

```diff
@@ -650,49 +669,12 @@ protected final void afterInternal(boolean afterClass) throws Exception {
         } finally {
             if (!success) {
                 // if we failed here that means that something broke horribly so we should clear all clusters
-                afterTestRule.forceFailure();
+                // TODO: just let the exception happen, WTF is all this horseshit
+                // afterTestRule.forceFailure();
             }
         }
     }
 
-    @Override
-    protected final AfterTestRule.Task afterTestTask() {
-        return new AfterTestRule.Task() {
-            @Override
-            public void onTestFailed () {
-                //we can't clear clusters after failure when using suite scoped tests, as we would need to call again
```

Comment:
since we don't use it here in elasticsearch, i'm getting it the hell out. if its needed somewhere else in a different home, someone can take care of that.

previous experiences in my opinion are that the tests here are slow, unreliable, and do not reproduce. I want to do the exact opposite of them.